draft-ietf-core-yang-cbor-latest.md

```diff
@@ -1681,36 +1716,40 @@ Author/Change controller:
 ## CoAP Content-Formats Registry
 
 This document adds the following Content-Format to the "CoAP Content-Formats",
-within the "Constrained RESTful Environments (CoRE) Parameters" registry.
-
-| Media Type                             | Content Coding | ID   | Reference |
-| application/yang-data+cbor; id=name    |                | TBD1 | RFC XXXX  |
+within the "Constrained RESTful Environments (CoRE) Parameters"
+registry, where TBD3 comes from the "Expert Review" 0-255 range and
+TBD1 and TBD2 come from the "IETF Review" 256-9999 range.
```

Comment:
Please don't put the requested ranges in the final RFC. (A note-to-be-removed-by-the-RFC-Editor is fine.)

Reply:
Now says "// RFC Ed.: please replace TBDx with assigned IDs, remove the
requested ranges, and remove this note.\\"